versioncontrol/versioncontrol.go

```diff
@@ -0,0 +1,10 @@
+package versioncontrol
+
+import "net/http"
+
+//Client holds the necessary items to make api calls to various version control providers
+//TODO Would be nice to attach repo urls and paths to this object at some point
```

Comment:
If this is remaining as a TODO, I would file it as an issue an remove this comment.